rules.py

```diff
@@ -18,9 +18,12 @@
 import argparse
 import io
 import os.path
+import hashlib
+from datetime import datetime
 import pycountry
 import requests
 from lxml import etree
+from github import Github
```

Comment:
Got the following error running the modified rules.py locally. Is it related to my python version? I'm using Python 2.7.6.

Traceback (most recent call last):
  File "rules.py", line 26, in <module>
    from github import Github
ImportError: No module named github